x-pack/plugins/fleet/server/routes/agent_policy/handlers.ts

```diff
@@ -108,8 +109,25 @@ export const createAgentPolicyHandler: RequestHandler<
   const esClient = context.core.elasticsearch.client.asInternalUser;
   const user = (await appContextService.getSecurity()?.authc.getCurrentUser(request)) || undefined;
   const withSysMonitoring = request.query.sys_monitoring ?? false;
-
+  // TODO set first default policy to is_default
+  // TODO installing these packages might take long (about 2 mins) which causes network timeout. this results in an error, and fails next agent policy creation request. Could this be improved by bundled default packages?
   try {
+    if (withSysMonitoring) {
```

Comment:
potential problem:
installing these packages might take long (about 2 mins) which causes network timeout. this results in an error, and fails next agent policy creation request. Could this be improved by bundled default packages?

Reply:
Bundled default packages aren't going to make much of a difference here based on my investigations in #110500. We'll likely need https://github.com/elastic/elasticsearch/issues/77505 to fix this.